istioctl/cmd/proxyconfig.go

```diff
@@ -946,6 +946,77 @@ func secretConfigCmd() *cobra.Command {
 	return secretConfigCmd
 }
 
+func rootCACompareConfigCmd() *cobra.Command {
+	var podName1, podName2, podNamespace1, podNamespace2 string
+
+	rootCACompareConfigCmd := &cobra.Command{
+		Use:   "rootca-comp [pod/]<name-1>[.<namespace-1>] [pod/]<name-2>[.<namespace-2>]",
```

Comment:
I think the command should be `rootca-cmp`? Just an idea from https://github.com/google/go-cmp

```suggestion
		Use:   "rootca-cmp [pod/]<name-1>[.<namespace-1>] [pod/]<name-2>[.<namespace-2>]",
```

Reply:
sure, that's a good idea!